prog/weaveproxy/main.go

```diff
@@ -15,6 +15,23 @@ var (
 	defaultListenAddrs = []string{"tcp://0.0.0.0:12375", "unix:///var/run/weave.sock"}
 )
 
+type listOpts struct {
+	values *[]string
```

Comment:
Because `ListVar` is handed a pointer to a variable, so the value of the variable can be modified in place.